vowels.py

```diff
@@ -0,0 +1,28 @@
+# Automation onboarding
+# Write a program to find the number of vowels and consonants in a given string.
+# Vowels: a, e, i, o, u
+import re
+
+def vowels(input):
+    #print (input)
+    # define a list to store all vowels and all consonants
+    vowelList = []
+    consonantList = []
+
+    # remove any special characters and space
+    input2 = re.sub('[^\w]+|_', '', input)
+
+
+    for i in range(len(input2)):
+        #print(input[i])
+        character = input[i]
+        if input2[i] == 'a' or input2[i] == 'e' or input2[i] == 'i' or input2[i] == 'o' or input2[i] == 'u':
```

Comment:
This works fine, you can also create a vowel array, and then the if statement will be simpler and smaller: 

```
vowels = ["a", "e", "i", "o", "u"]
if input2[i] in vowels:
    ......
```

Reply:
omg!! mind blowing!! Thank you!!